package.json

```diff
@@ -93,6 +98,12 @@
                 "mac": "cmd+b",
                 "when": "editorTextFocus && !editorReadonly && editorLangId == markdown"
             },
+            {
+                "command": "markdown.extension.editing.toggleQuote",
+                "key": "ctrl+.",
+                "mac": "cmd+.",
+                "when": "editorTextFocus && !editorReadonly && editorLangId == markdown"
+            },
```

Comment:
I prefer to not add a default key binding for this command as it is not very popular.